src/dex-ui/store/swapSlice/swapSlice.ts

```diff
@@ -155,70 +297,22 @@ const createSwapSlice: SwapSlice = (set, get): SwapStore => {
         false,
         SwapActionType.SEND_SWAP_TRANSACTION_TO_WALLET_STARTED
       );
-      /**
-       * Only L49A and L49B swaps are currently supported. The isTokenToTradeL49A and isTokenToTradeL49B booleans
-       * were created as a temporary work around to support the current swapToken Swap contract function logic.
-       * */
-      const isTokenToTradeL49A = tokenToTrade.symbol === TOKEN_A_SYMBOL;
-      const isTokenToTradeL49B = tokenToTrade.symbol === TOKEN_B_SYMBOL;
-      const tokenToTradeAccountId = TOKEN_SYMBOL_TO_ACCOUNT_ID.get(tokenToTrade.symbol) ?? "";
-      // const tokenToReceiveAccountId = TOKEN_SYMBOL_TO_ACCOUNT_ID.get(tokenToReceive.symbol) ?? "";
+      const tokenToTradeAccountId = tokenToTrade.tokenMeta.tokenId ?? "";
       const signingAccount = wallet.savedPairingData?.accountIds[0] ?? "";
-      const abstractSwapId = ContractId.fromString(SWAP_CONTRACT_ID);
+      const swapContractId = ContractId.fromString(tokenToTrade.tokenMeta.pairAccountId ?? "");
```

Comment:
We should move type conversations for `ContractId`, `AccountId`, `TokenId`, etc. into the HederaService layer. This will help keep our data conversions consistant across other functions using the HederaService functions.

Reply:
Done